apps/server/src/modules/school/api/school.uc.ts

```diff
@@ -32,6 +37,22 @@ export class SchoolUc {
 		return responseDto;
 	}
 
+	public async getSchoolSystems(schoolId: EntityId, userId: EntityId): Promise<SchoolSystemsResponse> {
+		const [school, user] = await Promise.all([
+			this.schoolService.getSchoolById(schoolId),
+			this.authorizationService.getUserWithPermissions(userId),
+		]);
+
+		const authContext = AuthorizationContextBuilder.read([Permission.SCHOOL_SYSTEM_VIEW]);
+		this.authorizationService.checkPermission(user, school, authContext);
+
+		const systems = await this.schoolService.getSchoolSystems(school);
```

Comment:
Das könnte/sollte auch in das Promise.all verschoben werden. Dadurch werden keine zusätzlichen kritischen Informationen preis gegeben.

Reply:
Wir haben da keine schule noch